src/main/kotlin/model/PrisonVisitsBooking.kt

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+
+class PrisonVisitsBooking private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    protected lateinit var frontend: Container
+    protected lateinit var backend: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "Prison Visits Booking",
+        "A service for booking a social visit to a prisoner in England or Wales"
+      )
+
+      frontend = system.addContainer("Frontend", "Public interface for booking a prison visit", "Ruby on Rails").apply {
+        setUrl("https://github.com/ministryofjustice/prison-visits-public")
+        Tags.WEB_BROWSER.addTo(this)
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      backend = system.addContainer("Backend", "API for the frontend and admin interface for staff to manage bookings", "Ruby on Rails").apply {
+        setUrl("https://github.com/ministryofjustice/prison-visits-2")
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      val zendesk = system.addContainer("Customer service", "Handles feedback tickets raised by staff and members of the public", "Zendesk").apply {
```

Comment:
📝 88377e6b6fb11cd300bce29516660fc7bda2bc09 has details on why it's a C4 container